recipes/visisipy/conda-forge.yml

```diff
@@ -0,0 +1,2 @@
+noarch_platforms:
```

Comment:
Why isn't this just a regular `noarch: python` package? I don't see any platform specific dependencies.

Reply:
Our general policy is that we just leave packages that are themselves `noarch: python` as such even if a dependency might have platform restrictions/is missing on a specific platform. It's ok if the CI fails for those platforms as long as that is the only cause. If you could remove the `conda-forge.yml` and the `__win`, that would be great.